iree/turbine/kernel/wave/codegen/read_write.py

```diff
@@ -377,6 +378,23 @@ def _linearize_memref(
     )
 
 
+def _get_splat_input(src: Optional[Value]) -> Optional[Value]:
+    """
+    If `src` is vector.splat result, return splat input, otherwise return None.
+    """
+    if src is None:
+        return None
+
+    try:

```

Comment:
Since this is not doing much in the except branch, how about replacing with getattr?

Reply:
done

Reply:
thanks